packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -268,6 +268,7 @@ abstract class FlameChartState<T extends FlameChart,
   @override
   void dispose() {
     zoomController.dispose();
+    focusNode.dispose();
```

Comment:
same question - why not use autoDisposeFocusNode?

Reply:
Good point, done!